web-wallet/src/lib/stores/networkStore.js

```diff
@@ -78,12 +51,23 @@ const getAccountSyncer = () => connect().then(() => new AccountSyncer(network));
 const getAddressSyncer = (options) =>
   connect().then(() => new AddressSyncer(network, options));
 
+/** @type {NetworkStoreServices["init"]} */
+async function init() {
+  const info = await network.node.info;
+
+  set({
+    ...initialState,
+    networkName: info.chain?.toString(),
```

Comment:
Why the optional chaining? Isn't `chain` supposed to always exist?

Reply:
It's interesting about that.  I wrote that line with Matteo (in it's original location in a top-level await) and the IDE auto-added the question marks as I selected the properties from the intellisense suggestions.  I didn't bother to change it at the time.